puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/foreman_infra.groovy

```diff
@@ -90,6 +90,21 @@ def runCicoJobsInParallel(jobs) {
     }
 }
 
+runCicoPipelines(project, version, config) {
```

Comment:
Added this function to generate the pipelines array with correct naming based on simple input.

Reply:
This is indeed what I had in mind.